docs/v8-node.md

```diff
@@ -101,3 +101,25 @@ Sentry.setupExpressErrorHandler(app);
 
 app.listen(3000);
 ```
+
+## Fastify
```

Comment:
we should probably write a guide for Fastify in dour docs at some point (probably not a blocker but is this something for #11064?)

Reply:
yes, once we update the docs for v8 we need to add guides for fastify, nest.js, and hapi, as well as updating the koa guides (and express, of course)! I'll add it to the issue, good point!